tweak(mobs): remove the multitask fun police

TimErmolt opened the pull request:
Вновь можно совершать подряд несколько действий (мультитаск) шприцом, то есть ввод реагентов или забор крови.

Далее чисто техническая инфа.

У `do_after()`, оказывается, вообще не было функционала возможности разрешения мультитаска (`can_multitask`). Я, конечно, добавил, билд компилится, и шприцы работают нормально, но всегда есть вероятность внезапного укуса в жопу откуда-то ещё из недр билда. Если вдруг чего возникнет, пофикшу, только дайте знать.

<details>
<summary>Чейнджлог</summary>

```yml
🆑TheUnknownOne & FloweYStyle
tweak: Теперь вновь можно совершать подряд действия шприцом ("закликивать"). Изменять количество вводимых/забираемых реагентов в шприце всё ещё можно.
/🆑
```

</details>

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашёл.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

TimErmolt committed:
tweak(mobs): remove the multitask fun police

Multitasking is now allowed for all `do_mob` and `do_after` actions. As it used to be.

TimErmolt committed:
ffs

i1yadobr commented:
Вот этот фикс мне больше нравится.

FelixArzt commented:
Не считаю мультитаск необходимым. Космонавтик-Шива это не то, жмите кнопочки по очереди.

FloweYStyle commented:
> Не считаю мультитаск необходимым. Космонавтик-Шива это не то, жмите кнопочки по очереди.

Ебитесь в сраку по очереди, как говорится. А то что в нынешних раундах врачей нет, тебе ни о чём не говорит видимо. За врачей **больно играть**. Ты не можешь практически нихуя сделать во время. Если у тебя больше одного пациента, а ты единственный врач на станции в силу очень частых обстоятельств, то у тебя нет ни единого шанса на спасение других. Ты не можешь быстро оперировать, ты не можешь удобно иньектировать в людей лекарства, у тебя забрали абсолютно все привычные тебе вещи в угоду... А в угоду чего блять? Форточку бы вы нахуй прикрыли, в чём была суть убийства мультитасков? В угоду реализма? На опухбее, где бегают клоуны и мимы, сжигают людей те же самые люди, станцию подрывают внезапный отряд оперативников с именами каклошвайн, яебалсобак и прочие кадры. Или там маги, ниндзи, это типо всё реалистично? Здесь этот реализм нахуй не нужен, а остальных причин я просто не наблюдаю убивать мультитаск. Помимо этого, убийство мультитаска также нанесло определённый урон духоты в сторону охраны и инженерного, с невозможностью быстро снимать одежду и ставить несколько стен. За что ты воюешь, Феликс? В чём суть, в чём твой путь?
Люди хотели бы узнать, почему ты сделал это решение, но ты им ответил ебаться с этим самостоятельно, как билд велел. А причины то какие, Феликс? Твоей хотелки недостаточно, чтобы что либо отменять, просто потому что ты считаешь это не необходимым. Я надеюсь ты прочитаешь это и подумаешь над своим решением дважды.

FelixArzt commented:
Лично играю только в меде, не вижу проблемы в том, чтобы у шприца выставить количество вкалываемого вещества, вот уменьшить количество времени для действия звучит как опция, которую с удовольствие одобрю, но миллион действий за секунду выглядит больше как баг, а не как фича. Будет ПР с уменьшением времени, которое необходимо для сварки/укола/т.п. - одобрю.

FelixArzt commented:
Ладно, обсудили вот это вот ваше мультитасковое.

Добавить мультитаск шприцам и все.

TimErmolt committed:
Revert "tweak(mobs): remove the multitask fun police"

This reverts commit dce706d9f5245aacae37e242ce6058ab0e281e3e.

TimErmolt committed:
Revert "ffs"

This reverts commit b9c99a8ebda6e5336005bc8fb30df6e58f7b5cb4.

TimErmolt commented:
Так-с.

TimErmolt committed:
tweak(syringes): enable multitasking

Enables multitasking for syringes (drawing blood/injecting stuff).
Also adds `can_multitask` to `do_after()` because apparently it wasn't a thing before????

TimErmolt commented:
Продублирую комментарием техническую информацию из описания:

У `do_after()`, оказывается, вообще не было функционала возможности разрешения мультитаска (`can_multitask`). Я, конечно, добавил, билд компилится, и шприцы работают нормально, но всегда есть вероятность внезапного укуса в жопу откуда-то ещё из недр билда. Если вдруг чего возникнет, пофикшу, только дайте знать.

А так, к мёржу готово. @FelixArzt
